src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -241,17 +246,12 @@ public BitbucketRepository getRepository() throws IOException, InterruptedExcept
         }
     }
 
-    public void deletePullRequestComment(String pullRequestId, String commentId) throws IOException, InterruptedException {
```

Comment:
Removing it as its not used anywhere in the code. Also searched jenkinsci and its not used by any plugins.